crates/modelardb_server/src/main.rs

```diff
@@ -165,6 +152,18 @@ fn main() -> Result<(), String> {
     // Setup CTRL+C handler.
     setup_ctrl_c_handler(&context, &runtime);
 
+    // Initialize storage manager with spilled buffers.
```

Comment:
It does bring up the question of when to use engine and when to use manager since we currently use both but we can probably just stick to what we currently use for now.

Reply:
[Storage engine](https://en.wikipedia.org/wiki/Database_engine) is the term commonly used for the component of a DBMS that reads and writes data.